spec.md

```diff
@@ -349,6 +349,8 @@ It is therefore RECOMMENDED that implementations avoid such strings.
 
 \* Fields with an asterisk are REQUIRED.
 
+Service Brokers MAY include additional fields within the response.
```

Comment:
In order to make clients that are generally useful in the presence of extensions, we will have to put an `extensions` object into each request.  If we allow extension fields to go directly into the namespace of the first-class spec fields, it will be much harder to write clients for the API.

Reply:
Why would it be harder to create a client w/o a wrapper? Granted, some languages (like go) don't handle extensions by default but it is possible to support them. I'm not necessarily pushing back on an "extension" wrapper, but I'm not in favor of having some language's limitations dictate what our spec-wise solution should be, so I'd like to better understand why this impact clients.

Reply:
We should at least make sure that there will be no conflicts in the future. For error responses we have this sentence: ` When adding new fields, Service Brokers SHOULD use a unique prefix for the field names to reduce the chances of conflict with future specification defined fields.`
We could make a more general statement regarding additional fields. For example, if we agree never to use certain characters (e.g. `.` or `:`) in field names in the spec then using field names with these characters in extension field names is safe.

Reply:
I added the guidance as you suggested. Not sure about the special char things yet...